plugins/auth/index.js

```diff
@@ -66,15 +67,26 @@ exports.register = (server, options, next) => {
     });
 
     /**
-     * Generates a jwt that is signed and has a 12h lifespan
+     * Generates a jwt that is signed
      * @method generateToken
-     * @param  {Object} profile Object from generateProfile
-     * @return {String}         Signed jwt that includes that profile
+     * @param  {Object}     profile     Object from generateProfile
+     * @param  {Boolean}    expires     True if the token should expire
+     * @return {String}                 Signed jwt that includes that profile
      */
-    server.expose('generateToken', profile => jwt.sign(profile, pluginOptions.jwtPrivateKey, {
-        algorithm: ALGORITHM,
-        expiresIn: EXPIRES_IN
-    }));
+    server.expose('generateToken', (profile, expires = true) => {
```

Comment:
This is gross and there's probably a better way to do this, please if somebody has an idea let me know.